datumaro/plugins/open_images_format.py

```diff
@@ -552,15 +553,25 @@ def _load_and_resize_mask(path, size):
         return resized.astype(bool)
 
 class OpenImagesImporter(Importer):
+    POSSIBLE_ANNOTATION_PATTERNS = (
+        OpenImagesPath.FULL_IMAGE_DESCRIPTION_FILE_NAME,
+        *OpenImagesPath.SUBSET_IMAGE_DESCRIPTION_FILE_PATTERNS,
+        '*' + OpenImagesPath.LABEL_DESCRIPTION_FILE_SUFFIX,
+        '*' + OpenImagesPath.BBOX_DESCRIPTION_FILE_SUFFIX,
+        '*' + OpenImagesPath.MASK_DESCRIPTION_FILE_SUFFIX,
+    )
+
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        with context.alternatives():
+            for pattern in cls.POSSIBLE_ANNOTATION_PATTERNS:
+                with context.alternative():
+                    context.require_file(
+                        f'{OpenImagesPath.ANNOTATIONS_DIR}/{pattern}')
```

Comment:
Is there a way to return confidence for a matched alternative?

Reply:
Do you mean to return different confidences for different alternatives?

I hadn't really considered this before (I have not encountered any cases where this would be necessary), but I think you can arrange the alternatives from most confident to least confident and then add `return <confidence>` to each.

Reply:
Yes, I thought it is what was meant by "returning different confidences for different alternatives". Maybe, the word "alternatives" doesn't fit exactly to the solution implemented, because it implies that the variants are mutually-exclusive.

Reply:
> Maybe, the word "alternatives" doesn't fit exactly to the solution implemented, because it implies that the variants are mutually-exclusive.

That's a fair point. After thinking about it, I decided to rename `FormatDetectionContext.alternatives` to `require_one_or_more`, which is a better name for several reasons:

* more clear what it does;
* consistent with the "requirement" terminology;
* visually distinct from `alternative`.

I kept `alternative` as it was, though, because I don't know what else to call it.

Reply:
How about `require_any`?

Reply:
> How about require_any?

Works for me. Renamed.

> Consider adding a confidence parameter to alternative().

I'll consider it when I actually need it.